sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/SessionCatalog.scala

```diff
@@ -170,6 +170,13 @@ class SessionCatalog(
     tableRelationCache.invalidate(key)
   }
 
+  /** Discards any cached table relation plans for the given table identifier. */
+  private[sql] def invalidateCachedTable(name: TableIdentifier): Unit = {
```

Comment:
`SessionCatalog` is already internal, we don't need "private[sql]" here

Reply:
I just followed existing methods there:
https://github.com/apache/spark/blob/157b72ac9fa0057d5fd6d7ed52a6c4b22ebd1dfc/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/SessionCatalog.scala#L1652

Reply:
why not follow `def invalidateCachedTable(key: QualifiedTableName): Unit`?

Reply:
@cloud-fan In general, how do you figure out that some class is an internal or not?

Reply:
if you can't see it in https://spark.apache.org/docs/latest/api/scala/org/apache/spark/index.html , it's internal